tests/unit/redirects/test_models.py

```diff
@@ -44,6 +44,16 @@ def test_redirect_all_target_objects():
         RedirectFactory(**kwargs)
 
 
+def test_exception_when_no_field_name_found():
+    class FakeTarget:
+        name = ""
+
+    redirect = RedirectFactory(project=ProjectFactory(), old_url="/test/")
+    redirect.targets = lambda: [FakeTarget(), FakeTarget()]
```

Comment:
thought: The fact we have to resort to monkeypatching the object to test this is a smell.

Unless someone knows a way to create a valid `Redirect` which doesn't have a matching field name (which seems unlikely because these are baked into the class).

Reply:
I agree, this seems like an unhelpful test. I actually think the `# pragma: no branch` is maybe more expressive in this instance (we're explicitly saying that we haven't anticipated this happening).

Reply:
I removed this test and restored the `pragma` here.